### ISSUE TITLE
[Settings - Line Height] : Label 'Line Height' is not associated with the 'Reset' button.

### ISSUE THREAD

**Issue by Saiteja341** [microsoft/terminal#15158]
### Windows Terminal version

1.18.902.0

### Windows build number

25324.1011

### Other Software

Test Environment:
OS: Windows 11 Version Dev (OS Build 25324.1011)
App: Windows Terminal Preview
Screen Reader: Narrator

### Steps to reproduce

Repro Steps:

1.Open Windows Terminal.
2.Open Setting page using 'Ctrl + ,' keys.
3.Now navigate to the 'Advanced' button present under 'Default Profile'.
4. Now navigate to the 'Line Height's reset' button and observe the issue.

User Experience:
People with visual impairments who rely on the screen readers are unable to know the button's description.

Guideline Link:
https://www.w3.org/TR/WCAG21/#info-and-relationships

Attachments:
[Label 'Line Height' is not associated with the 'Reset' button..zip](https://github.com/microsoft/terminal/files/11197431/Label.Line.Height.is.not.associated.with.the.Reset.button.zip)

### Expected Behavior

Label 'Line Height' should be associated with the 'Reset' button.

### Actual Behavior

Label 'Line Height' is not associated with the 'Reset' button.
